.github/workflows/publish.js.yml

```diff
@@ -44,9 +44,15 @@ jobs:
           -scheme WebDriverAgentRunner \
           -destination generic/platform=iOS \
           CODE_SIGNING_ALLOWED=NO ARCHS=arm64
-    - name: Creating a zip of WebDriverAgentRunner-Runner.app for iOS
+    - name: Creating a zip of WebDriverAgentRunner-Runner.app for iOS after removing test frameworks
       run: |
         pushd appium_wda_ios/Build/Products/Debug-iphoneos
+        rm -rf WebDriverAgentRunner-Runner.app/Frameworks/XCTAutomationSupport.framework
```

Comment:
can it be simplified to  rm -rf WebDriverAgentRunner-Runner.app/Frameworks/XC*.framework ?

Reply:
same below

Reply:
Yea, it can. I'd have liked to remove paths explicitly